### ISSUE TITLE
Update wiki

### ISSUE THREAD

**Issue by GPaolo** [sferes2/sferes2#91]
Hi, I think the wiki needs an update.
In the modifier part there is only the dummy listed, not the diversity or the novelty.

**Comment by jbmouret** [comment]
You are right! We are actually moving the documentation to: http://sferes2.github.io/sferes2/ 

(but the issue that you are pointing out is probably there too, as this is based on the wiki).
